gnome/gnucash/Portfile

```diff
@@ -39,8 +39,8 @@ checksums         rmd160  25ad9822130e641240418d753a0092c98815688b \
                   sha256  b60772c276a63639964d1e24c795fbb9a7c0c6c45959168fe5cdfe08eb796f16 \
                   size    14166587
 
-patchfiles-append gnucash-fix-python-include-dir.diff
-
+patchfiles-append patch-python-include-dirs-typo-fix.diff
+patchfiles-append patch-gnucash-remove-test-failing-to-build.diff
```

Comment:
Ruh Roh! I renamed a patchfile due to "port lint nitpick" saying I should. Well, too much trouble to move it to that other commit now, so it stays here!